src/v2/ion/i18nUtils.js

```diff
@@ -30,6 +30,7 @@ const SECURITY_QUESTION_PREFIXES = [
 const I18N_OVERRIDE_MAPPINGS = {
   'identify.identifier': 'primaryauth.username.placeholder',
   'select-authenticator-unlock-account.identifier': 'primaryauth.username.placeholder',
+  'unlock-account.identifier': 'primaryauth.username.placeholder',
```

Comment:
Which string does this override?

Reply:
This maps the `unlock-account` remediation name to the "Username" label above the identifier field so that label can be localized by our translation bundles. Before identifier-first support, we weren't using the `unlock-account` remediation so no mapping existed